src/module/transaction/component/input/AssetAmountInput/AssetAmountInput.tsx

```diff
@@ -0,0 +1,23 @@
+import { TextFieldProps } from "module/common/component/input/TextField/TextField.types";
+import { Asset } from "module/wallet/component/input/WalletAssetSelect/WalletAssetSelect.types";
+import { AssetType } from "module/wallet/wallet.types";
+import NEARAmountInput from "./NEARAmountInput/NEARAmountInput";
+import NftAmountInput from "./NftAmountInput/NftAmountInput";
+import TokenAmountInput from "./TokenAmountInput/TokenAmountInput";
+
+export interface AssetAmountInputProps extends Omit<TextFieldProps, "keyboardType" | "validators"> {
+    asset: Asset;
+    index?: number;
+}
+
+const AssetAmountInput = ({ asset, index = 0, onChange, value, ...rest }: AssetAmountInputProps) => {
+    return (
+        <>
+            {asset.type === AssetType.TOKEN && <NEARAmountInput onChange={onChange} value={value} index={index} {...rest} />}
+            {asset.type === AssetType.FT && <TokenAmountInput onChange={onChange} value={value} ft={asset.ft!} {...rest} />}
```

Comment:
Es podrien mostrar de manera més xula, no? Per exemple mostrant el icono, o en el cas del NFT la imatge/nom o algo així, potser podem fer una pr millorant això, què penses?

Reply:
Mmm no ho veig massa, maybe bastant més andavant. Amb el suffix q es mostra "NEAR" o "USDT" i ja es veu força bé. I ficar el nom/image al TextField seria too much crec tmb.
Nse igual canviar el suffix per l'icono igual sii però hi ha altres coses q rentan més de moment

Reply:
![image](https://user-images.githubusercontent.com/52425638/212290046-fae1167a-3858-40ff-9333-0cd69dba2f2c.png)